node/service/src/relay_chain_selection.rs

```diff
@@ -338,16 +338,27 @@ impl<B> SelectChain<PolkadotBlock> for SelectRelayChain<B>
 				.map_err(|e| ConsensusError::Other(Box::new(e)))?
 			{
 				// No approved ancestors means target hash is maximal vote.
-				None => (target_hash, target_number),
-				Some((s_h, s_n)) => (s_h, s_n),
+				None => (target_hash, target_number, Vec::new()),
+				Some((s_h, s_n, s_bd)) => (s_h, s_n, s_bd),
 			}
 		};
 
+		// 3. Constrain according to disputes:
+		let (tx, rx) = oneshot::channel();
+		overseer.send_msg(DisputeCoordinatorMessage::DetermineUndisputedChain{
+				base_number: subchain_number,
+				block_descriptions: subchain_block_descriptions,
+				tx,
+			},
+			std::any::type_name::<Self>(),
```

Comment:
I missed where this is coming from - some sender string in the new communication model?

Reply:
This adds some additional information for certain error cases, Credit goes to @ordian